text/0002-sarif-result-schema.md

```diff
@@ -0,0 +1,1939 @@
+      */
+      minimumRequiredLocalizedDataSemanticVersion?: string;
+
+      /**
+      * The name of the tool component.
+      */
+      name: string;
+
+      /**
+      * An array of reportingDescriptor objects relevant to the notifications related to the configuration and runtime
+      * execution of the tool component.
+      */
+      notifications?: ReportingDescriptor[];
+
+      /**
+      * The organization or company that produced the tool component.
+      */
+      organization?: string;
+
+      /**
+      * A product suite to which the tool component belongs.
+      */
+      product?: string;
+
+      /**
+      * A localizable string containing the name of the suite of products to which the tool component belongs.
+      */
+      productSuite?: string;
+
+      /**
+      * A string specifying the UTC date (and optionally, the time) of the component's release.
+      */
+      releaseDateUtc?: string;
+
+      /**
+      * An array of reportingDescriptor objects relevant to the analysis performed by the tool component.
+      */
+      rules?: ReportingDescriptor[];
+
+      /**
+      * The tool component version in the format specified by Semantic Versioning 2.0.
+      */
+      semanticVersion?: string;
+
+      /**
+      * A brief description of the tool component.
+      */
+      shortDescription?: MultiformatMessageString;
+
+      /**
+      * The tool component version, in whatever format the component natively provides.
+      */
+      version?: string;
+
+      /**
+      * Key/value pairs that provide additional information about the tool component.
+      */
+      properties?: PropertyBag;
+  }
+
+  /**
+  * The runtime environment of the analysis tool run.
+  */
+  export interface Invocation {
+      /**
+      * An array of strings, containing in order the command line arguments passed to the tool from the operating
+      * system.
+      *
+      * CHECKUP: args go heree 
+      */
+      arguments?: string[];
+
+      /**
+      * The command line used to invoke the tool.
+      */
+      commandLine?: string;
+
+      /**
+      * The Coordinated Universal Time (UTC) date and time at which the run ended. See "Date/time properties" in the
+      * SARIF spec for the required format.
+      * 
+      * CHECKUP: this property along with startTimeUtc will contain timing info
+      */
+      endTimeUtc?: string;
+
+      /**
+      * The environment variables associated with the analysis tool process, expressed as key/value pairs.
+      * 
+      * CHECKUP: this property will contain the flags, config, configHash 
+      */
+      environmentVariables?: { [key: string]: string };
+
+      /**
+      * Specifies whether the tool's execution completed successfully.
+      */
+      executionSuccessful: boolean;
+
+      /**
+      * The process exit code.
+      */
+      exitCode?: number;
+
+      /**
+      * The machine that hosted the analysis tool run.
+      */
+      machine?: string;
```

Comment:
I think https://oclif.io/docs/config contains this info.

Reply:
```An invocation object MAY contain a property named machine whose value is a string containing the name of the machine on which the tool was run.

``` 

It seems like oclif has the name of the platform and the shell, but I don't see the name of the machine?

Reply:
Got it.